koap/src/jsMain/kotlin/jsBridge.kt

```diff
@@ -0,0 +1,30 @@
+package com.juul.koap
+
+import com.juul.koap.adapter.toJson
+import com.juul.koap.adapter.toMessage
+
+@JsModule("object-hash")
+@JsNonModule
+external fun <T> hash(value: T): String
```

Comment:
Why is this needed?

Reply:
It is not. I had removed the `ObjectHasher` and the external dependency for hashing on the JS side, this should have been removed in that commit. Removed.